tests/post/steps/test_monitoring.py

```diff
@@ -252,7 +252,8 @@ def check_deployed_rules(host, prometheus_api):
             # For now, we only need alerting rules
             if rule['type'] == "alerting":
                 message = rule['annotations'].get('message') or \
-                    rule['annotations'].get('summary')
+                    rule['annotations'].get('summary') or \
+                    rule['annotations'].get('description')
```

Comment:
If we update this test, we should likely also update the tool which generates the list of alerts we're comparing with: https://github.com/scality/metalk8s/blob/development/2.7/tools/rule_extractor/rule_extractor.py#L127-L136

Reply:
(and maybe re-run the tool)

Reply:
Done